tests/Fixer/NamespaceNotation/NoLeadingNamespaceWhitespaceFixerTest.php

```diff
@@ -43,100 +43,114 @@ public static function provideFixCases(): iterable
             $manySpaces[] = 'namespace Test'.$i.';';
         }
 
-        return [
-            // with newline
-            ["<?php\nnamespace Test1;"],
-            ["<?php\n\nnamespace Test2;"],
-            [
-                "<?php\nnamespace Test3;",
-                "<?php\n namespace Test3;",
-            ],
-            // without newline
-            ['<?php namespace Test4;'],
-            [
-                '<?php namespace Test5;',
-                '<?php  namespace Test5;',
-            ],
-            // multiple namespaces with newline
-            [
-                '<?php
+        yield // with newline
+        ["<?php\nnamespace Test1;"];
```

Comment:
the comment should be placed above the `yield` as in the original

Reply:
@mvorisek fixed in https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7193